druid-shell/Cargo.toml

```diff
@@ -21,7 +21,7 @@ log = "0.4.8"
 lazy_static = "1.0"
 time = "0.2.7"
 cfg-if = "0.1.10"
-kurbo = "0.5.9"
+kurbo = "0.5.10"
```

Comment:
Should be 0.5.11?

Reply:
yea, this is outdated I was just closing.